molgenis/molgenis-emx2
# fix(directory): add study table to staging template

What are the main changes you did:
- Added studies table + studies column in Collections to the molgenis.csv of the biobank_directory staging template
- Included demo studies data

how to test:
- Goto the preview, create a new database using biobank_directory_staging template, including demo-data => check that the Studies table is there, including some demo data. Also the Collections table contains now a Studies column linked to the Studies table

todo:
- [nvt] updated docs in case of new feature
- [nvt] added/updated testplan to include a test for this fix, including ref to bug using # notation
